swagger.yaml

```diff
@@ -3786,6 +3786,72 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/clusters/{cluster_id}/installer-ui-settings:
```

Comment:
Why not just `ui-settings`? Why add `installer`? The whole application is an installer.